region/lowernorfair/east/Plowerhouse Room.json

```diff
@@ -608,6 +630,25 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [2, 2],
+      "name": "G-Mode Setup - Get Hit By Zebbo",
+      "entranceCondition": {
+        "comeInNormally": {}
+      },
+      "requires": [
+        {"heatFrames": 375}
```

Comment:
I found doing a loop around the left holtz as the way I would do this.  That gives a room time of 150frames.  Otherwise I think you would synchronize the two holtzes but I find that just as tricky.